pkg/util/resource.go

```diff
@@ -275,3 +275,13 @@ func MinInt64(a, b int64) int64 {
 	}
 	return b
 }
+
+// IsNodeReady checks whether the node condition is ready.
+func IsNodeReady(node *corev1.Node) bool {
+	for _, condition := range node.Status.Conditions {
+		if condition.Type == corev1.NodeReady {
+			return condition.Status == corev1.ConditionTrue
+		}
+	}
+	return false
+}
```

Comment:
1. Please move this function to `pkg/util/helper/node.go`
2. Consider rename to `NodeReady()`